src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6223,16 +6223,16 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ImplicitRangeIndexerWithName" xml:space="preserve">
     <value>Invocation of implicit Range Indexer cannot name the argument.</value>
   </data>
-  <data name="ERR_TypelessNewIllegalTargetType" xml:space="preserve">
+  <data name="ERR_ImplicitObjectCreationIllegalTargetType" xml:space="preserve">
     <value>The type '{0}' may not be used as the target type of new()</value>
   </data>
-  <data name="ERR_TypelessNewNotValid" xml:space="preserve">
+  <data name="ERR_ImplicitObjectCreationNotValid" xml:space="preserve">
     <value>Use of new() is not valid in this context</value>
   </data>
-  <data name="ERR_TypelessNewNoTargetType" xml:space="preserve">
+  <data name="ERR_ImplicitObjectCreationNoTargetType" xml:space="preserve">
     <value>There is no target type for '{0}'</value>
   </data>
-  <data name="IDS_FeatureTargetTypedObjectCreation" xml:space="preserve">
+  <data name="IDS_FeatureImplicitObjectCreation" xml:space="preserve">
     <value>target-typed object creation</value>
```

Comment:
Should we change this string as well?

Reply:
I don't think so. That lines up with other "target-typed" features.